src/Screens/PushNotificationsScreen.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  getInitialNotification,
+  isRegisteredForNotifications,
+  onNotificationOpened,
+  onNotificationReceived,
+  requestNotificationsPermissions,
+} from '@lifeomic/react-native-sdk';
+import { useEffect } from 'react';
+import { Notifications } from 'react-native-notifications';
+import { Pressable, StyleSheet, Text } from 'react-native';
+
+const styles = StyleSheet.create({
+  button: {
+    paddingVertical: 12,
+    paddingHorizontal: 32,
+    margin: 100,
+    borderRadius: 4,
+    elevation: 3,
+    backgroundColor: '#4287f5',
+  },
+  text: {
+    fontSize: 16,
+    lineHeight: 21,
+    letterSpacing: 0.25,
+    color: 'white',
+    textAlign: 'center',
+  },
+});
+
+export const PushNotificationsScreen = () => {
+  const sendLocalNotification = () => {
+    const notification = Notifications.postLocalNotification({
+      body: 'Local notification!',
+      title: 'Local Notification Title',
+      sound: 'chime.aiff',
+      //@ts-ignore
+      silent: false,
+      category: 'SOME_CATEGORY',
+      userInfo: {},
+    });
+  };
+
+  useEffect(() => {
+    requestNotificationsPermissions(({ deviceToken, denied, error }) => {
+      return { deviceToken, denied, error };
```

Comment:
for a lot of these return statements, im not exactly sure what we want to do with these values. or if we even want to return them at all and instead log them.

Reply:
I imagine we want to provide an example implementation. Such as (pseudo code)
```typescript
if (error || denied) {
  setError(error) // See the SDK for showing a <Toast /> message
}

// Register device with application 
api.post('/v1/device-endpoints'. { deviceToken, provider: 'device platform, e.g. apns or gcm', application: 'my-app' }
```